polaris-tokens/src/token-groups/color.ts

```diff
@@ -920,7 +920,7 @@ export const color: {
     description: '',
   },
   'color-bg-transparent-experimental': {
-    value: colorsExperimental.gray[16](),
+    value: colorsExperimental.gray[16]('0.001'),
```

Comment:
```suggestion
    value: colorsExperimental.gray[16]('0'),
```

Nice catch! It should actually be 0% opacity. The `.001` is a Figma hack. You need a bg color for things like shadows to show up and setting the opacity really low is a work around. I might find another solution though to avoid the confusion.

Reply:
Done, thnx @sarahill 🙏